public/src/components/channelManagement/bannerTests/bannerTestEditor.tsx

```diff
@@ -23,7 +23,10 @@ import TestVariantsSplitEditor from '../testVariantsSplitEditor';
 import { useStyles } from '../helpers/testEditorStyles';
 
 const copyHasTemplate = (content: BannerContent, template: string): boolean =>
-  (content.heading && content.heading.includes(template)) || content.messageText.includes(template);
+  (content.heading && content.heading.includes(template)) ||
+  (content.paragraphs != null && content.paragraphs[0].includes(template)) ||
```

Comment:
Not happy with this line - I think that we need to be testing for the template variable against all paragraphs joined into a string?

Reply:
Just check all paras in the array -
`(content.paragraphs && content.paragraphs.some(para => para.includes(template))`

Reply:
Done